int32 RobotState_Stop = 0
int32 RobotState_Remote = 1
int32 RobotState_Autonomous = 2
int32 RobotState_FollowSidewalk = 3